### PR TITLE
Windows kubectl tests

### PR DESCRIPTION
**Pull request by cynepco3hahue**
- create windows VM via kubectl
- delete windows VM via kubectl

Signed-off-by: Lukianov Artyom <alukiano@redhat.com>

### CODE REVIEW

**Comment by cynepco3hahue** [comment]
@rmohr I added a timeout to event watcher, to prevent stuck of VM start function.

File: tests/utils.go [hunk 1]
```diff
@@ -1050,3 +1058,40 @@ func SkipIfNoWindowsImage(virtClient kubecli.KubevirtClient) {
...
 }
+
+func SkipIfNoKubectl() {
+	if KubeVirtKubectlPath == "" {
+		Skip("Skip test that requires kubectl binary")
+	}
+}
+
+func RunKubectlCommand(args ...string) error {
+	kubeconfig := flag.Lookup("kubeconfig").Value
```
[the remark this answers is not in the document]
Comment (cynepco3hahue) [reply tests/utils.go]: @rmohr I thought about it, I also thought that we need to pass `--server` parameter to `kubectl`, but I see that we do not pass `--server` parameter to functional tests, so I omitted it.
```bash
${TESTS_OUT_DIR}/tests.test -kubeconfig=${kubeconfig} -tag=${docker_tag} -prefix=${functest_docker_prefix} -test.timeout 60m ${FUNC_TEST_ARGS}
```
Also `server` variable is important when do you have different contexts under the same configuration file but in our case, we have the configuration file with only one context.
What do you think?

[the remark this answers is not in the document]
Comment (rmohr) [reply tests/utils.go]: It is valid for the test binary to use `-kubeconfig` and `-master`. `-server` is not defined here. We can switch to `-server` later if we want.

File: tests/windows_test.go [hunk 2]
```diff
@@ -217,4 +221,52 @@ var _ = Describe("Windows VM", func() {
...
+
+		AfterEach(func() {
+			if yamlFile != "" {
+				err = os.Remove(yamlFile)
+				Expect(err).ToNot(HaveOccurred())
+				yamlFile = ""
+			}
+		})
+
+		It("should success to start a vm", func() {
```
Comment (rmohr) [inline tests/windows_test.go]: success/succeed

File: tests/windows_test.go [hunk 3]
```diff
@@ -217,4 +221,52 @@ var _ = Describe("Windows VM", func() {
...
+			By("Starting the vm via kubectl command")
+			err = tests.RunKubectlCommand("create", "-f", yamlFile)
+			Expect(err).ToNot(HaveOccurred())
+
+			tests.WaitForSuccessfulVMStartWithTimeout(windowsVm, 120)
+		})
+
+		It("should success to stop a vm", func() {
```
Comment (rmohr) [inline tests/windows_test.go]: success/succeed

**Pull request merged**